content/minfraud/sandbox-environment.mdx

```diff
@@ -0,0 +1,79 @@
+how to set up a Sandbox account on our Knowledge
+Base.](https://support.maxmind.com/hc/en-us/articles/8604168846363)
+
+Once you have your Sandbox account set up and activated, follow the steps below.
+
+### Generate a license key for your Sandbox account
+
+In order to test the minFraud services in the Sandbox environment, you will need
+to generate a license key from your Sandbox account. [Learn how to generate a
+license key on our Knowledge
+Base.](https://support.maxmind.com/hc/en-us/articles/4407111582235-Generate-a-License-Key)
+The process is the same for your Sandbox account, but you must login to your
+Sandbox account [(direct
+link)](https://sandbox.maxmind.com/en/accounts/current/license-key) rather than
+your regular MaxMind account.
+
+### Build your integration
+
+Consult our [quick start guide](/minfraud/evaluate-a-transaction?lang=en) or our
+[list of client
+APIs](/minfraud/evaluate-a-transaction?lang=en#links-to-maxmind-client-apis) to
+develop your integration.
+
+### Point your integration to the Sandbox service endpoints
+
+Use the service endpoints for the Sandbox version of the web services, as
+listed in the table below:
+
+| Sandbox Service | HTTP Method | Endpoint                                             |
+| --------------- | ----------- | ---------------------------------------------------- |
+| Score           | `POST`      | `https://sandbox.maxmind.com/minfraud/v2.0/score`    |
+| Insights        | `POST`      | `https://sandbox.maxmind.com/minfraud/v2.0/insights` |
+| Factors         | `POST`      | `https://sandbox.maxmind.com/minfraud/v2.0/factors`  |
+
+Remember to use the license key generated by your Sandbox account to
+authenticate your requests.
+
+### Test data for the Sandbox minFraud service
+
+You may only submit approved test data to the Sandbox version of the minFraud
+services. At this time, the only approved test transaction data are the three IP
+addresses listed below. You can submit transactions to Sandbox minFraud services
+with any of the following IP addresses as the [`/device/ip_address`](/minfraud/api-documentation/requests?lang=en#schema--request--device__ip_address)
+input. You should recieve a minFraud response in which all fields are blank
+except for the IP risk score output ([`/ip_address/risk`](/minfraud/api-documentation/responses?lang=en#schema--response--ip-address__risk)),
+which will contain the expected response below.
+
+| Test IP Address Input | Test IP Risk Score Output |
+| --------------------- | ------------------------- |
+| `146.243.121.22`      | 75                        |
+| `152.216.7.110`       | 0.01                      |
+| `23.10.87.103`        | 0.01                      |
+
+You should not submit other transaction data inputs along with the IP address,
+and you should never submit actual customer transaction data to the Sandbox
+version of the minFraud services.
+
+## Test Legacy versions of minFraud using the Sandbox
```

Comment:
It is not clear to me why this is a level 2 heading, but everything above is a level 3. I think they should probably all be level 2. This is also contributing to one of the build failures.